engine/src/main/java/org/terasology/logic/players/ThirdPersonRemoteClientSystem.java

```diff
@@ -121,39 +121,30 @@ public void ensureHeldItemIsMountedOnLoad(OnChangedComponent event, EntityRef cl
 
     @Command(shortDescription = "Sets the held item mount point translation for remote characters")
     public void setRemotePlayersHeldItemMountPointTranslations(@CommandParam("x") float x, @CommandParam("y") float y, @CommandParam("z") float z) {
-        for (EntityRef remotePlayer : entityManager.getEntitiesWith(CharacterComponent.class, PlayerCharacterComponent.class)) {
-            if (relatesToLocalPlayer(remotePlayer)) {
-                RemotePersonHeldItemMountPointComponent newComponent = remotePlayer.getComponent(RemotePersonHeldItemMountPointComponent.class);
-                if (newComponent != null) {
-                    newComponent.translate = new Vector3f(x, y, z);
-                    ensureClientSideEntityOnHeldItemMountPoint(OnActivatedComponent.newInstance(), remotePlayer, newComponent);
-                }
+        for (EntityRef remotePlayer : entityManager.getEntitiesWith(RemotePersonHeldItemMountPointComponent.class)) {
+            RemotePersonHeldItemMountPointComponent remoteMountPointComponent = remotePlayer.getComponent(RemotePersonHeldItemMountPointComponent.class);
+            if (remoteMountPointComponent != null) {
```

Comment:
Very minor thing is that you *technically* don't need this check.
The `entityManager.getEntitiesWith` ensures that they will have the component and thus you don't need to check if it is null.

Reply:
Right I missed this, thanks 👍